### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by netshaman** [Nibre/MotherVR#108]
After starting alien Vr, either in sit position or stand up position, i have a black screen in the HMD when moving or using doors etc.
It seems to  appear randomly .
Though i don't try to move out of the boudaries, don't know what iti is about, but Oculus home says " bad localisation problem " or such a message, weird, it is the first time i have this issue ingame.
Some clues ?
I will retry as at the moment a new version of Oculus software is downloading ...

**Comment by spogulis** [comment]
Did you try pressing right controller's menu button and pressing "Reset View" in Oculus Home? I did encounter black screen issues when hiding in closets or under tables and such.

**Comment by netshaman** [comment]
Yes , but  it don't work.
There is something that deactivate the inner displays*, so i only hear the sound .
*The displays are switched off.

**Comment by GladiTek** [comment]
Yeah I have exactly the same problem and it hinders me from enjoying this otherwise great mod.
